package.json

```diff
@@ -64,6 +64,7 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start": "./bin/start.js",
     "pretest": "standard",
-    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'"
+    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'",
+    "postinstall": "node ./set-start.js"
```

Comment:
I think you are right, the`postinstall` script is run _only_ after the package is installed that defines the `postinstall` script. I’ve run a few tests myself and it seems like it. Makes sense anyway :) Thanks for checking!